addons/crm/models/crm_lead.py

```diff
@@ -1984,6 +1983,9 @@ def _pls_get_lead_pls_values(self, domain=[]):
         """
         leads_values_dict = OrderedDict()
         pls_fields = ["stage_id", "team_id"] + self._pls_get_safe_fields()
+        use_tags = 'tag_ids' in pls_fields
```

Comment:
use_tags = pls_fields.pop('tag_ids', False) ? Not sure what is the type of pls_fields. A comment to explain tags are managed separately would be welcomed.

Reply:
Hagrid with the comment to add. But indeed, pls_field is a list and not a dict. so pop works with index -> And that make an ugly thing. I would keep it like this.